### ISSUE TITLE
Schedule broken in Develop branch

### ISSUE THREAD

**Issue by mdaneri** [Badgerati/Pode#1395]
### Describe the Bug
The last Pull #1393 has broken the Pode Schedule 

### Steps To Reproduce
run the `examples\Schedule.ps1`

### Expected Behavior
```
Name                           Value
----                           -----
Name                           predefined
OnStart                        False
NextTriggerTime
EndTime
StartTime
UsingVariables                 {[SubExpressions, System.Object[]], [NewNameWithDollar, $__using_message], [OldName, $using:message], [Value, Hello, world!]…}
LastTriggerTime                9/19/2024 6:22:00 AM
Script                         …
Count                          2
CronsRaw                       {@minutely}
Arguments                      {}
Limit                          2
Completed                      True
Crons                          {[DayOfMonth, System.Collections.Hashtable], [Random, False], [Minute, System.Collections.Hashtable], [Hour, System.Collections.Hashtable]…}

Last: <something here>
Next: <and here>
Message1:
Message2:
Hello, there!
double cron
Hello, there!
```
 
### Actual Behavior
```
Name                           Value
----                           -----
Name                           predefined
OnStart                        False
NextTriggerTime
EndTime
StartTime
UsingVariables                 {[SubExpressions, System.Object[]], [NewNameWithDollar, $__using_message], [OldName, $using:message], [Value, Hello, world!]…}
LastTriggerTime                9/19/2024 6:22:00 AM
Script                         …
Count                          2
CronsRaw                       {@minutely}
Arguments                      {}
Limit                          2
Completed                      True
Crons                          {[DayOfMonth, System.Collections.Hashtable], [Random, False], [Minute, System.Collections.Hashtable], [Hour, System.Collections.Hashtable]…}

Last:
Next:
Message1:
Message2:
Hello, there!
double cron
Hello, there!
```

**Comment by Badgerati** [comment]
I've found why it worked for myself, and now doesn't in the `develop` branch.

For Schedules/Tasks the `param` names in the supplied scriptblock have to be explicitly named, and it's just a by-product of the scriptblock originally running directly as runspaces rather than being invoked via a wrapping scriptblock - and I've left it as is for backwards compatibility support.

This means if you want to use the event object, you need a parameter explicitly called `$Event`:
```powershell
Add-PodeSchedule -Name 'predefined' -Cron '@minutely' -Limit 2 -ScriptBlock {
    param($Event, $Message1, $Message2)
    $using:message | Out-Default
    Get-PodeSchedule -Name 'predefined' | Out-Default
    "Last: $($Event.Sender.LastTriggerTime)" | Out-Default
    "Next: $($Event.Sender.NextTriggerTime)" | Out-Default
    "Message1: $($Message1)" | Out-Default
    "Message2: $($Message2)" | Out-Default
}
```

It seems to stem from #1333, and one we missed, but in there the parameter name was changed:
```powershell
Add-PodeSchedule -Name 'predefined' -Cron '@minutely' -Limit 2 -ScriptBlock {
    param($TriggeredEvent, $Message1, $Message2)
    $using:message | Out-Default
    Get-PodeSchedule -Name 'predefined' | Out-Default
    "Last: $($TriggeredEvent.Sender.LastTriggerTime)" | Out-Default
    "Next: $($TriggeredEvent.Sender.NextTriggerTime)" | Out-Default
    "Message1: $($Message1)" | Out-Default
    "Message2: $($Message2)" | Out-Default
}
```

Changing `$TriggeredEvent` back to `$Event` fixes it.

I can fix it, though not sure on how to get a test in there for it. I'll have a think.

**Comment by mdaneri** [comment]
I can fix it here

**Comment by mdaneri** [comment]
Fixed in #1384 
I also created a pester test for that
